tests/exit-status.sh

```diff
@@ -1,43 +1,58 @@
 #!/usr/bin/env bash
-# Grab the execution directory
-SCRIPT_DIR="$( dirname "$( readlink -f "${BASH_SOURCE[0]}" )")"
+
+# Grab the execution directory.
```

Comment:
Point taken about these, the whole construct appears unused and lines 2-9 can go.

Reply:
Okay, sounds good.